src/Components/DefaultLayout/PickersHeader.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import { Nav } from 'reactstrap';
+
+class PickersHeader extends Component {
```

Comment:
The purpose of this Component is Breadcrumb instead of PickersHeader. Also please expose Props so other can pass content of breadcrumb, better have an example page to show how to use this.